tests/multi_net/asyncio_tls_server_client.py

```diff
@@ -0,0 +1,76 @@
+# Test uasyncio TCP server and client using start_server() and open_connection()
+
+try:
+    import os
+    import asyncio
+    import ssl
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+PORT = 8000
+
+# These are test certificates. See tests/README.md for details.
+cert = cafile = "multi_net/rsa_cert.der"
+key = "multi_net/rsa_key.der"
+
+try:
+    os.stat(cafile)
+    os.stat(key)
+except OSError:
+    print("SKIP")
+    raise SystemExit
+
+
+async def handle_connection(reader, writer):
+    # Test that peername exists (but don't check its value, it changes)
+    writer.get_extra_info("peername")
```

Comment:
I don't think this line is needed, the `get_extra_info()` method is tested already in an existing test.